### PR TITLE
Cumulative Graphs

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes
- addresses #666 
- remove some magic strings

## Todo:
- [ ] people endpoint

## Checklist
- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/dashboard/DashboardItem.js [hunk 1]
```diff
@@ -24,7 +24,15 @@ import { dashboardColorNames, dashboardColors } from 'lib/colors'
 import { useLongPress } from 'lib/hooks/useLongPress'
 
 const typeMap = {
-    ActionsLineGraph: {
+    ActionsLineGraphLinear: {
```
Comment (EDsCODE) [inline frontend/src/scenes/dashboard/DashboardItem.js:27]: cypress test picked this bug up!

**Review by timgl — COMMENTED** [review]
Trying to add this to the dashboard and navigating to that dashboard results in a whiteout and this error:
image

I also wonder whether this should show that it's a cumulative graph to avoid confusion:
image

File: frontend/src/lib/constants.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const ACTIONS_LINE_GRAPH_LINEAR = 'ActionsLineGraph'
+export const ACTIONS_LINE_GRAPH_CUMULATIVE = 'ActionsLineGraphCumulative'
+
+export const STICKINESS = 'Stickiness'
+
+export const LINEAR_CHART_LABEL = 'Linear Chart'
```
Comment (timgl) [inline frontend/src/lib/constants.js:6]: I think it'd look better if this was just `'Linear'` but up to you

**Review by timgl — APPROVED** [review]
One NIT, otherwise all good

**Pull request merged**